operator/cmd/mesh/operator_test.go

```diff
@@ -76,7 +78,7 @@ func TestOperatorInit(t *testing.T) {
 			tag:               "1.2.3",
 			operatorNamespace: "operator-test-namespace",
 			watchedNamespaces: "istio-test-namespace1,istio-test-namespace2",
-			manifestsPath:     string(snapshotCharts),
+			manifestsPath:     string(liveCharts),
```

Comment:
I think `liveCharts` read data from the `manifests` directory. So in that case this test will not read YAML from `testdata` directory?

Reply:
updated the snapshotCharts and use that.